aws_eventbridge/eventbridge-single-region.yaml

```
@@ -0,0 +1,125 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Datadog AWS Cloudtrail Eventbright Logs Integration
```

```suggestion
Description: Datadog AWS Cloudtrail Eventbridge Logs Integration
```
I think this is a typo?